packages/vhd-format-lwt/vhd-format-lwt.0.12.3/opam

```diff
@@ -7,15 +7,14 @@ homepage: "https://github.com/mirage/ocaml-vhd"
 doc: "https://mirage.github.io/ocaml-vhd/"
 bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
 depends: [
-  "ocaml" {>= "4.02.3"}
+  "ocaml" {>= "4.02.3" & < "5.0.0"}
   "cstruct" {< "6.1.0"}
   "lwt" {>= "3.2.0"}
   "mirage-block" {>= "2.0.1"}
   "ounit2" {with-test}
   "vhd-format"
-  "io-page-unix" {with-test}
   "dune" {>= "1.0"}
-  "io-page" {with-test & >= "2.4.0"}
+  "io-page" {with-test & = "2.4.0"}
```

Comment:
We try to avoid equality constraints when it is not a coherent set of packages (e.g. from the same repository or same developer). Would that work instead?
```suggestion
  "io-page" {with-test & >= "2.4.0" & < "3.0.0"}
```